README.md

```diff
@@ -0,0 +1,71 @@
+# Preface
+
+This assume you have installed codius, if you havent please do so first. If you are using the community installer, this install guide will be added soon on that script.
+
+# How to install codiusd-gui:
+
+This will install the GUI and once sucessfully started it will run on 127.0.0.1 on port 3300. You can SSH 
+redirect in, or you can add an NGinx config
+
+```bash
+cd /usr/lib/node_modules
+git clone https://github.com/codius/codiusd-gui.git
+cd /usr/lib/node_modules/codiusd-gui/
+npm install
+cp ./extra/systemd/codiusd-gui.service /etc/systemd/system/codiusd-gui.service
+systemctl daemon-reload
+systemctl enable codiusd-gui
+systemctl start codiusd-gui
+```
+
+# Add Nginx Config
+To keep it simple we use the same template as with Codius. In this example you will see a 'allow 127.0.0.1;'. You must add another line with your trusted IP address to gain access.
+
+```
+map $http_upgrade $connection_upgrade {
+  default upgrade;
+  '' $http_connection;
+}
+
+server {
+    listen 444 ssl;
```

Comment:
It would be beneficial to tell the reader that nginx will start listening on port 444, which will have to be explicitly included in the url (e.g. https://codius.example.com:443) to be reached unlike 443 and 80, which are implicitly routed to for HTTPS and HTTP requests.